src/app/products/utils/trade-path.ts

```diff
@@ -0,0 +1,8 @@
+export const buildEarnTradePath = (symbol: string) =>
+  `/earn?buy=${symbol}&sell=ETH&network=1`
```

Comment:
nice, remember tho that icUSD will start on Base if relevant

Reply:
@janndriessen good call - added support to specify the chain for a token: https://github.com/IndexCoop/index-app/pull/1653/files#diff-851083ccb97f87f6f4e3b4075d3ba8700bd411f221a6991cd4c1129db5b0199dR4